src/Engines/webgpuEngine.ts

```diff
@@ -2084,7 +2086,8 @@ export class WebGPUEngine extends Engine {
             gpuTextureWrapper = this._textureHelper.createGPUTextureForInternalTexture(texture);
         }
 
-        createImageBitmap(video).then((bitmap) => {
+        const engine = EngineStore.LastCreatedEngine;
+        engine && engine.createImageBitmap(video).then((bitmap) => {

```

Comment:
This is the engine class right?

```suggestion
        this.createImageBitmap(video).then((bitmap) => {
```

Reply:
..ah, Gary already pointed to it!